docs/implementation.md

```diff
@@ -32,3 +32,14 @@ There are two primary entry points for middleware:
 
 1. `robot.listenerMiddleware` - registers a new piece of middleware in a global array
 2. `middleware.execute` - executes all registered middleware in order
+
+## Persistence
+
+Hubot has an in-memory key-value store exposed as `robot.brain` that can be used to store and retrieve data.
+Furthermore, `hubot-redis-brain` enables persistence across Hubot restarts.
```

Comment:
I would say that there are hubot scripts which enables persistence, and list hubot-redis-brain as one example.